packages/launchpad/src/signingcosmosclient.ts

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import { AminoMsg, Coin, makeSignDoc, OfflineAminoSigner, StdFee } from "@cosmjs/amino";
+import {} from "@cosmjs/amino/build/signdoc";
```

Comment:
What happened here?

Reply:
Thanks, this is a remnant of autocomplete when I'd forgotten to export something.